api/cds.proto

```diff
@@ -63,8 +63,7 @@ message CircuitBreakers {
 
 message Cluster {
   // Supplies the name of the cluster which must be unique across all clusters.
-  // The cluster name is used when emitting statistics. The cluster name can be
-  // at most 60 characters long, and must not contain :.
+  // Any : in the cluster name will be converted to _ when emitting statistics.
```

Comment:
FWIW this is the kind of place in which we would cross link to the CLI option. Might be worth putting a doc TODO in here.